bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx110-en.html

```diff
@@ -0,0 +1,77 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew') }}
```

Comment:
Question (non-blocking): I'm not sure when we started overriding the default WNP style sheet (I see a few other instances of it for previous releases), but is it intentional? I'm undecided if this is actually easier to maintain this way, or harder longer term (versus each WNP release having a single style sheet). Thoughts?

Reply:
You're right, if this style sheet is included we don't need the imports, or if we leave this out then we do need the imports. We certainly don't need both. I think it was a one-time oversight that has perpetuated through save-as.